switchmate.py

```diff
@@ -6,16 +6,19 @@
 
 Usage:
 	./switchmate.py scan
+	./switchmate.py status
 	./switchmate.py <mac_address> auth
 	./switchmate.py <mac_address> <auth_key> switch [on | off]
 	./switchmate.py -h | --help
 """
 
-from __future__ import print_function
```

Comment:
@brianpeiris Out of curiosity, why do you choose to use the Python 3 print function instead of Python 2?